### PR TITLE
fixed choice of Ksig parameter in ci.in

### PR DESCRIPTION
**Pull request by b0ndarev**

### CODE REVIEW

**Review by ccheung93 — COMMENTED** [review]
wouldn't this change affect all-order when trying to read effective 2-electron radial integrals from SCRC.CON?

**Comment by b0ndarev** [comment]
As far as I got, the logic is as if SCRC.CON is presented then, set Ksig to 2, if not to 1. So it shouldn't effect

**Comment by ccheung93** [comment]
In the case of all-order, I believe we use Ksig=2. So this change would not work for all-order. I think we can add a condition if include_qed, we set Ksig to 1, otherwise leave as 2.

File: py-lib/ci.py [hunk 1]
```diff
@@ -257,7 +257,7 @@ def move_conf_inp(conf_dir, root_dir, parity, run_codes, include_lsj, write_hij)
             if os.path.isfile('basis/SCRC.CON'):
                 f.write('Ksig = 2 \n')
             else:
-                f.write('Ksig = 2 \n')
+                f.write('Ksig = 1 \n')
```
Comment (ccheung93) [inline py-lib/ci.py:260]: now that i can see the surrounding code, yes this is fine.

**Review by ccheung93 — APPROVED** [review]
now that i can see the surrounding code, yes this is what is supposed to be intended.

**Comment by ccheung93** [comment]
now that i can see the surrounding code, yes this is what is supposed to
be intended. you can merge when you get the chance

**Pull request merged**